doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -1,10 +1,10 @@
 FORMAT: 1A
 HOST: http://orion.lab.fiware.org
-TITLE: FIWARE-NGSI v2 Specification
-DATE: 30 September 2015
-VERSION: 2.0-latest
-PREVIOUS_VERSION: <placeholder>
-APIARY_PROJECT: fiware-ngsi-v2
+TITLE: FIWARE-NGSI v2 Specification (WIP)
+DATE: to be fixed upon next RC production
+VERSION: latest
+PREVIOUS_VERSION: stable
+APIARY_PROJECT: orioncontextbroker
```

Comment:
It corresponds to:

http://docs.orioncontextbroker.apiary.io

Note that the project previously to this PR

http://docs.fiware-ngsi-v2.apiary.io

doesn't exist.